tsconfig.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Try to order you props by alphabetical order, it's easier to read. At least the `compilerOptions` ones.

Reply:
done